projects/storefrontapp-e2e-cypress/cypress/support/require-logged-in.commands.ts

```diff
@@ -89,7 +89,6 @@ Cypress.Commands.add(
       account.registrationData.email ||
       generateMail(account.user, options.freshUserOnTestRefresh);
 
-    cy.server();
```

Comment:
We need to keep this until we adjust all e2e using it to cy.intercept

Reply:
I put it back